neuraxle/pipeline.py

```diff
@@ -23,13 +23,17 @@
 import os
 from abc import ABC, abstractmethod
 from copy import copy
-from typing import Any, Tuple
+from typing import Any, Tuple, List, Iterable
 
+from conv import convolved_1d
 from joblib import load, dump
 
-from neuraxle.base import BaseStep, TruncableSteps, NamedTupleList, ResumableStepMixin, DataContainer
+from neuraxle.base import BaseStep, TruncableSteps, NamedTupleList, ResumableStepMixin, DataContainer, NonFittableMixin, \
+    NonTransformableMixin, Barrier
 from neuraxle.checkpoints import BaseCheckpointStep
 
+BARRIER_STEP_NAME = 'Barrier'
```

Comment:
Please don't do that. Either do `Barrier.__class__.__name__`, or i'd still prfer `Barrier.__class__` without as string. 

You resolved my previous comment without fix and I'm repeating. We don't need this constant.

Reply:
dead code